src/ng/directive/ngModel.js

```diff
@@ -1075,13 +1093,17 @@ var ngModelDirective = ['$rootScope', function($rootScope) {
             });
           }
 
+          function setTouched() {
+            modelCtrl.$setTouched();
+          }
+
           element.on('blur', function(ev) {
             if (modelCtrl.$touched) return;
 
             if ($rootScope.$$phase) {
-              scope.$evalAsync(modelCtrl.$setTouched);
```

Comment:
Is it possible to add un-minified only checks to aid in the transition for these methods?

`console.assert(this instanceof NgModelController, 'Some link to the breaking change message');`

It's not worth these checks in the minified build imo since those should be the production assets, but it may be worth it in the debug/unminified builds.